manual/manual/refman/exten.etex

```diff
@@ -2044,29 +2036,30 @@ the inline record as a pseudo-value, but the record cannot escape the
 scope of the binding and can only be used with the dot-notation to
 extract or modify fields or to build new constructor values.
 
-\begin{verbatim}
+\begin{caml_example*}{verbatim}
 type t =
   | Point of {width: int; mutable x: float; mutable y: float}
-  | ...
+  | Other
 
 let v = Point {width = 10; x = 0.; y = 0.}
 
 let scale l = function
   | Point p -> Point {p with x = l *. p.x; y = l *. p.y}
-  | ....
+  | Other -> Other
```

Comment:
Was this change by choice or due to the lack of attribute on case? If it is the second, using 
`| Other[@ellipsis.start] -> Other[@ellipsis stop]` would work after a small fix that I can send as a sub-PR.

Reply:
I think I will make a test PR for ellipsis attributes in general.